selfapi.js

```diff
@@ -6,6 +6,11 @@ var https = require('https');
 var nodepath = require('path');
 var url = require('url');
 
+var options = {
+    jsonStringifyReplacer: null,
+    jsonStringifySpaces: 2
```

Comment:
Nit: This file uses 2 space indentation, but this parameter has 4 spaces in front of it 😋 quite ironic if you think about it.